### ISSUE TITLE
HardwareGrowler - Notifications issues with Mavericks (using Mac OSX notifications instead)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tazounet/growl#615]
```
What steps will reproduce the problem?
1. Just install HardwareGrowler and Growl (it's working fine) on Mavericks
2.
3.

What is the expected output?
All notifications using Smoke notifications.
What do you see instead?
 I have a mix of notifications using Mac OSX style
What version of the product are you using? On what operating system?
Latest one. Mavericks 10.9

Please provide any additional information below.
I love Growl. I have been using it for years.

Yesterday I upgraded my Mac OSX to Mavericks. Growl is working fine but 
Hardware Growler is not.

Basically I have this:

When I press Caps On – I have a notification with Smoke style (Great a I love 
that)
But sometimes, I am receiving notifications using the Mac OSX style 

I didn’t have this issue with Mountain Lion.
```

Original issue reported on code.google.com by `n...@netofrombrazil.com` on 23 Oct 2013 at 2:51
- Merged into: #617
